lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -742,6 +749,9 @@ abstract class DefaultTextEditingStrategy implements TextEditingStrategy {
     this._inputConfiguration = inputConfig;
 
     _domElement = inputConfig.inputType.createDomElement();
+    if (inputConfig.readOnly) {
```

Comment:
Can this value change later? If so, do we handle the change correctly?

Reply:
Configurations are only sent once when the connection is established.